conf/application.conf

```diff
@@ -63,8 +63,6 @@ application.langs="en"
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
 play.http.router=prod.Routes
 
-appUrl = "http://localhost:9669/"
-
 play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
```

Comment:
on line 70, MicroserviceModule is outdated, use `uk.gov.hmrc.play.bootstrap.backend.BackendModule` instead
on line 71, do we still need those filters?

Reply:
also, on line 29 there's something from the Play 2.3 era, which doesn't exist anymore (it doesn't even give a deprecation message at this point)

Reply:
two more outdatednesses: line 42, `play.crypto.secret` -> `play.http.secret.key` and line 52, `application.langs` -> `play.i18n.langs`